src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,140 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private class Node<T> {
+        private Node<T> prev;
+        private T value;
+        private Node<T> next;
+
+        public Node(Node<T> prev, T value, Node<T> next) {
+            this.prev = prev;
+            this.value = value;
+            this.next = next;
+        }
+    }
+
+    private static final int START_ARRAY_LENGTH = 10;
+    private Node [] elements;
```

Comment:
in a linked list elements shouldn't be saved in an array. Instead, the list should only know about the first node and the last node. other nodes should be accessed by iterating from the end of the list